### PR TITLE
Fix/#32

### LINKED ISSUE #32: Bug In The Text Of The Selected Language  /  Error En El texto Del Idioma Seleccionado
### Explanation:
The text Select language should be closer to the left side of the box, the font is not correct, we want the modification to be done and the text to be closer to the left side of the box, so that the user can see the text with a better presentation, this error occurs in all three language options. (it is presented in all fields and sources)

**Actual result:** The text is not centered. The font is not correct.

**Expected result:** That the text is centered. That the font is correct.

*Screen size:*  Full screen.

### Screenshots:
Original: 
image

Our: 
image
image

### Checklist:
This issue can be closed when the following tasks are completed:
 
 - [ ] When the texts look more centered.
 - [ ] When the font is correct.

### References:

Original: https://www.figma.com/proto/9hPU5c4waRbBOd8msfEnKa/Search_by_time_github?node-id=2%3A8&scaling=scale-down&page-id=0%3A1
Our: https://github.com/Azordev/search-by-time-github

Spanish Version:

### Explicación:
El texto Seleccionar idioma debe estar más cerca del lado izquierdo del cuadro, la fuente no es correcta, queremos que se realice la modificación y que el texto esté más cerca del lado izquierdo del cuadro, para que el usuario pueda ver el texto. Con una mejor presentación, este error ocurre en las tres opciones de idioma. (presentado en todos los campos y fuentes)

**Resultado real:** El texto no está centrado, la fuente no es correcta.

**Resultado esperado:** Que el texto esté centrado. Que la fuente sea correcta.

*Tamaño de pantalla:* Pantalla completa.

### Capturas de pantalla:
Original:
imagen

Nuestro:
imagen
imagen

### Lista de Verificación:
Este problema se puede cerrar cuando se completen las siguientes tareas:
 
 - [ ] Cuando los textos se vean más centrados.
 - [ ] Cuando la fuente  sea correcta.

### Referencias:
Original: https://www.figma.com/proto/9hPU5c4waRbBOd8msfEnKa/Search_by_time_github?node-id=2%3A8&scaling=scale-down&page-id=0%3A1
Nuestro: https://github.com/Azordev/search-by-time-github

### PR DESCRIPTION
**Pull request by mdrianislam0or1**
Issue Title:  Bug in select language text #32 

Fix the bug in select language text

fix32

### CODE REVIEW

**Comment by Israel-Laguan** [comment]
I notice now it moved to the other side hehe 

I suggest you use flex, "text-align: center;" or similar to center the text in the div

File: src/css/style.css [hunk 1]
```diff
@@ -67,7 +67,7 @@ body {
   border: 2px solid #c4c4c4;
   box-sizing: border-box;
   border-radius: 10px;
-  padding-left: 35px;
```
Comment (heliomar-pena) [inline src/css/style.css:70]: I think this padding is the reason why the text is not centered, please try removing it or adding equal padding to both sides and the text will be centered.

Comment (mdrianislam0or1) [reply src/css/style.css:70]: there already use text-align:center; bt the result is same.i used equal padding to both sides.

**Comment by mdrianislam0or1** [comment]
f32 1

File: src/css/style.css [hunk 2]
```diff
@@ -67,7 +67,8 @@ body {
...
+  padding-right: 30px;
+  padding-left: 30px;
```
Comment (heliomar-pena) [inline src/css/style.css:71]:
```suggestion
  padding: 0 30px;
```

**Review by heliomar-pena — APPROVED** [review]
I left you a little suggestion

**Review by Israel-Laguan — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

**Pull request merged**